src/util/EntityToStateMapper.js

```diff
@@ -153,9 +153,11 @@ const getHtmlFieldType = (fieldType: EntityFieldType): HtmlFieldType => {
     case 'LONG':
       return 'number'
     case 'TEXT':
+      return 'text-area'
```

Comment:
Looking at current molgenis forms, should TEXT type be normal 'text' input type with more then 255 characters validation?

Reply:
agree